ext/cl_khr_external_memory.asciidoc

```diff
@@ -211,6 +211,7 @@ If {CL_MEM_DEVICE_HANDLE_LIST_KHR} is not specified as part of _properties_, the
 
 The properties used to create a buffer or image from an external memory handle are described by related extensions.
 When a buffer or image is created from an external memory handle, the _flags_ used to specify usage information for the buffer or image must not include {CL_MEM_USE_HOST_PTR}, {CL_MEM_ALLOC_HOST_PTR}, or {CL_MEM_COPY_HOST_PTR}, and the _host_ptr_ argument must be `NULL`.
+For images created from an external memory handle, implementations will acquire information about image attributes such as format and layout at the time of creation and continue to apply those attributes for the lifetime of the image object.
```

Comment:
Suggest s/will/may/ as acquiring the information is not mandatory. The second part of the sentence could be tweaked accordingly, maybe:

>When images are created from an external memory handle, implementations may acquire information about image attributes such as format and layout at the time of creation. When such information is acquired at image creation time, it is used as  for the lifetime of the image object.

Reply:
I agree, and have made the requested change.

Thanks,
Joshua Kelly, Qualcomm